packages/website/src/components/config.ts

```diff
@@ -0,0 +1,6 @@
+export const detailTabs = [
+  { value: false as const, label: 'Errors' },
```

Comment:
[Nitpick] [Style] Maybe extract out a type for each element and make this whole variable an array of that?

```suggestion
export const detailTabs: DetailTab[] = [
  { value: false, label: 'Errors' },
```

(more posting out of curiosity for your style preferences than anything else, definitely nowhere near a blocker)

Reply:
generally if type is never going to be reused I prefer to use `as const` in this context, and if type is or may be reused I'm creating types